app/src/main/res/layout/fragment_data.xml

```diff
@@ -1,132 +1,248 @@
         android:layout_height="wrap_content"
-        android:layout_below="@+id/fragment_data_located"
-        android:orientation="horizontal"
-        android:paddingLeft="@dimen/padding_forty_eight"
-        android:paddingTop="@dimen/padding_five"
-        android:paddingRight="@dimen/padding_forty_eight"
-        android:paddingBottom="@dimen/padding_five">
-
-       	<TextView
-	    	android:layout_width="0dp"
-	        android:layout_height="@dimen/height_forty"
-	        android:textSize="@dimen/text_size_seventeen"
-	        android:layout_weight="3"
-	        android:textColor="@color/black"
-	        android:layout_marginTop="@dimen/margin_twenty"
-	        android:text="@string/to_sync"
-        />
-       	
-       	<TextView
-       	    android:id="@+id/fragmentDataToSyncValue"
-	    	android:layout_width="0dp"
-	        android:layout_height="@dimen/height_forty"
-	        android:textSize="@dimen/text_size_seventeen"
-	        android:layout_weight="1"
-	        android:gravity="right"
-	        android:textColor="@color/black"
-	        android:layout_marginTop="@dimen/margin_twenty"
-        />
-	    
-	</LinearLayout>	        
-
-	    <Button
-	        android:id="@+id/fragmentDataSyncButton"
-	    	android:layout_width="match_parent"
-	        android:layout_height="wrap_content"
-	        android:layout_margin="@dimen/activity_vertical_margin"
-	        android:text="@string/sync"
-			style="@style/NewButtonStyle"
-	        android:layout_alignParentBottom="true"/>
-
-      
-</RelativeLayout>
+        android:layout_below="@+id/tree_data_card_view"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="16dp"
+        android:layout_marginEnd="16dp"
+        app:cardBackgroundColor="@color/white"
+        app:cardCornerRadius="20dp"
+        app:cardElevation="10dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/fragment_data_payments_label">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/payment_data_layout"
+            android:layout_width="match_parent"
+            android:layout_height="120dp"
+            android:paddingLeft="@dimen/sixteen_dp"
+            android:paddingTop="@dimen/padding_five"
+            android:paddingRight="@dimen/sixteen_dp"
+            android:paddingBottom="@dimen/padding_five">
+
+            <androidx.constraintlayout.widget.Guideline
+                android:id="@+id/paymentDataGuideline"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:orientation="vertical"
+                app:layout_constraintGuide_begin="243dp" />
+
+            <TextView
+                android:id="@+id/paymentPendingLabel"
+                android:layout_width="0dp"
+                android:layout_height="@dimen/height_forty"
```

Comment:
It's great that you put this in a style, but can we rename it to something that doesn't mentioned it's value? If we ever change the internal value, we'd need to refactor all usages of the style.

Reply:
Are you referring to ```@dimen/height_forty```?  Do you want to rename it to something like ```@dimen/labelTextHeight``` ?